package.json

```diff
@@ -52,11 +59,11 @@
             "eslint"
         ],
         "**/src/**": [
-            "sfdx-lwc-jest --skipApiVersionCheck -- --bail --findRelatedTests"
+            "jest"
```

Comment:
Any reason why you removed the extra `--bail --findRelatedTests` flags from the command?
I would rather keep them.

Reply:
Good catch. Added the flags.